src/tss2-esys/esys_tr.c

```diff
@@ -509,3 +509,48 @@ Esys_TRSess_SetAttributes(ESYS_CONTEXT * esys_context, ESYS_TR esys_handle,
          sessionAttributes & ~mask) | (flags & mask);
     return TSS2_RC_SUCCESS;
 }
+
+/** Retrieve the TPM nonce of an Esys_TR session object.
+ *
+ * Some operations (i.e. Esys_PolicySigned) require the nonce returned by the
+ * TPM during Esys_StartauthSession. This function provides this nonce to the
+ * caller.
+ * @param esys_context [in,out] The ESYS_CONTEXT.
+ * @param esys_handle [in,out] The ESYS_TRsess for which to retrieve the nonce.
+ * @param nonceTPM [out] The nonce of the object (callee-allocated; use free()).
```

Comment:
s/use free/user free/

Reply:
this was actually meant as a verb to use free().